packages/appstore/src/assets/svgs/wallet/index.tsx

```diff
@@ -95,6 +95,11 @@ import WechatPayDark from 'Assets/svgs/wallet/wechat-pay-dark.svg';
 import WechatPayLight from 'Assets/svgs/wallet/wechat-pay-light.svg';
 import ZingpayDark from 'Assets/svgs/wallet/zingpay-dark.svg';
 import ZingpayLight from 'Assets/svgs/wallet/zingpay-light.svg';
+import DerivGo from 'Assets/svgs/wallet/ic-brand-derivgo.svg';
```

Comment:
we already have all of this icons in the components package, isn't it better to `import {Icon} from '@deriv/components'` in the `packages/appstore/src/components/platform-launcher/platform-launcher.tsx` and use it instead of `WalletIcon`?

Reply:
@mahdiyeh-fs  the plugin we use for  SVG bundling in  component folder having issue with SVG with gradient property.
But the one we using for wallet folder doesn't have any issue with gradient SVG's that is why we are using it here now.
if we can sort out the issues in the future we can use the same folder.